### ISSUE TITLE
[1.12.2] Crash upon loading new chunks (java.lang.NoClassDefFoundError: net/minecraft/client/particle/Particle) 

### ISSUE THREAD

**Issue by aking4125** [noobanidus/Lootr#211]
Hello,

Upon exploring some new chunks regardless of dimension. The server crashes with the following error but doesn't create a crash report file even though the log says it does:

```
[Server thread/ERROR] [FML]: Exception caught during firing event net.minecraftforge.fml.common.gameevent.TickEvent$ServerTickEvent@22f45d14:
java.lang.NoClassDefFoundError: net/minecraft/client/particle/Particle
	at com.hbm.blocks.test.TestObjTester.createNewTileEntity(TestObjTester.java:40) ~[TestObjTester.class:1.9.8]
	at net.minecraft.block.Block.createTileEntity(Block.java:1274) ~[aow.class:?]
	at noobanidus.mods.lootr.config.ConfigManager.addUnsafeReplacement(ConfigManager.java:332) ~[ConfigManager.class:?]
	at noobanidus.mods.lootr.config.ConfigManager.replacement(ConfigManager.java:359) ~[ConfigManager.class:?]
	at noobanidus.mods.lootr.block.tile.TileTicker.serverTick(TileTicker.java:98) ~[TileTicker.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_1402_TileTicker_serverTick_ServerTickEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) [EventBus.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:266) [FMLCommonHandler.class:?]
	at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:712) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_382]
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.particle.Particle
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_382]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_382]
	... 12 more
Caused by: net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerException: Exception in class transformer net.minecraftforge.fml.common.asm.transformers.SideTransformer@35cec305 from coremod FMLCorePlugin
	at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:260) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:279) ~[launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_382]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_382]
	... 12 more
Caused by: java.lang.RuntimeException: Attempted to load class btf for invalid side SERVER
	at net.minecraftforge.fml.common.asm.transformers.SideTransformer.transform(SideTransformer.java:62) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:256) ~[forge-1.12.2-14.23.5.2860.jar:?]
	at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:279) ~[launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_382]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_382]
	... 12 more
```

[debug-1.log.gz](https://github.com/Alcatergit/Hbm-s-Nuclear-Tech-GIT/files/12397187/debug-1.log.gz)

Any ideas? Only guess is related to lootr and HBM.

https://github.com/Alcatergit/Hbm-s-Nuclear-Tech-GIT/issues/220

**Comment by embeddedt** [comment]
This looks like an issue with HBM; their tile entity is accessing a client-only class on the server.

**Comment by aking4125** [comment]
Yeah, so it seems. Though blacklisting the HBM modid in lootr does nothing. 

But blacklisting the entire dimension, in this case the overworld does actually prevents the crash in new chunks as all chest are no longer lootr chest.

We did report it also to HBM, it is a strange issue.

**Comment by embeddedt** [comment]
I can patch Lootr to not outright crash here, although there is still the potential for some other mod to crash in the same way if it tries to reference that HBM tile entity.

**Comment by aking4125** [comment]
Yeah. I did refer the issue on HBMs side though no response as of yet.

I'm trying to blacklist a few more mod IDs in addition to HBM. Dunno if I either got the modid wrong or its not working.

Here is the config on the server we're trying as we're testing different modids (Had to change the .cfg to .txt cause github doesn't like .cfg files)

[lootr.txt](https://github.com/noobanidus/Lootr/files/12398596/lootr.txt)

**Comment by embeddedt** [comment]
I think the original crash should be fixed as of 0.6.2.

**Comment by aking4125** [comment]
Saw the update to Lootr, will report back after updating.
